pkg/api/v1/types.go

```diff
@@ -2450,6 +2450,16 @@ type PodSpec struct {
 	// +patchMergeKey=IP
 	// +patchStrategy=merge
 	HostAliases []HostAlias `json:"hostMappings,omitempty" patchStrategy:"merge" patchMergeKey:"IP" protobuf:"bytes,23,rep,name=hostMappings"`
+	// If specified, indicates the pod's priority. "SYSTEM" is a special keyword
+	// which indicates the highest priority. Any other name must be defined by a
+	// PriorityClass object with that name.
+	// If not specified, the pod priority will be zero.
+	// +optional
```

Comment:
No, they are placed correctly, but if they are confusing and you have a better suggestion, I will gladly fix them. `Priority` should never be set by users.